superbench/config/default.yaml

```diff
@@ -164,22 +166,22 @@ superbench:
         - gpt2-large
       parameters:
         <<: *common_model_config
-        batch_size: 4
+        batch_size: 1
```

Comment:
pls set it in common_model_config and remove it here

Reply:
ok, updated